lib/helpers/environment.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Helper functions pretaining to the environment domain
```

Comment:
Typo here: 
```suggestion
 * Helper functions pertaining to the environment domain
```

Reply:
Thank you for spotting this typo. I have fixed it in the latest commit.